### ISSUE TITLE
ACC does not convert empty monitor receive string into AS3 declaration

### ISSUE THREAD

**Issue by mnaumiuk** [f5devcentral/f5-automation-config-converter#75]
### Environment
 * ACC Version: 1.17
 * AS3 Version: 3.33.0
 * BIG-IP Version: 14.1.4.4

### Summary
ACC does not convert empty HTTP monitor receive string ("recv none") into AS3 declaration, so AS3 will take the default one.
But the default AS3 receive string for HTTP monitor is “HTTP/1.”:
https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/schema-reference.html?highlight=delete%20tenant#monitor-http
This leads to health monitor errors after deploying the AS3 declaration.

Original monitor config ("recv none"):
```
ltm monitor http /Common/acc_test_http_monitor {
    adaptive disabled
    defaults-from /Common/http
    interval 5
    ip-dscp 0
    recv none
    recv-disable none
    send "GET /\r\n"
    time-until-up 0
    timeout 16
}
```

AS3 declaration (no receive string declared):
```
    "class": "ADC",
    "schemaVersion": "3.33.0",
    "id": "urn:uuid:fb23f83c-ece3-45e1-999a-0a69626f6419",
    "label": "Converted Declaration",
    "remark": "Generated by Automation Config Converter",
    "Common": {
....
            "acc_test_http_monitor": {
                "adaptive": false,
                "dscp": 0,
                "send": "GET /\\r\\n",
                "class": "Monitor",
                "monitorType": "http"
            }
        }
    }
}
```

Config after deploying AS3 ("recv HTTP/1."):
```
ltm monitor http /Common/Shared/acc_test_http_monitor {
    adaptive disabled
    adaptive-limit 1000
    adaptive-sampling-timespan 180
    defaults-from /Common/http
    destination *:*
    interval 5
    ip-dscp 0
    recv HTTP/1.
    recv-disable none
    send "GET /\r\n"
    time-until-up 0
    timeout 16
}

```

### Steps To Reproduce
1. Create custom HTTP monitor with defaults from parent HTTP monitor
2. assign the monitor to a pool with HTTP service (custom HTTP health monitor should mark pool UP)
2. convert config to AS3
3. Deploy AS3
4. custom HTTP health monitor will mark pool DOWN

### Expected Behavior
Because of the difference between BigIP defaults and AS3 defaults, ACC should add an empty HTTP monitor receive string into the AS3 declaration:
`recv none` --->` "recevie": ""`

### Actual Behavior
Empty receive string for HTTP monitor is omitted in AS3 declaration that causes using the default AS3 non-empty value ("HTTP/1.")

**Comment by mdditt2000** [comment]
Jira CHARON-568 for PM tracking

**Comment by mdditt2000** [comment]
tmsh is set to none, ACC will follow. When TMSH is set to none, ACC will not output that property. ACC is working correctly.  Closing github issues

**Comment by mdditt2000** [comment]
Revisiting
```
ltm monitor http /Common/acc_test_http_monitor {
    adaptive disabled
    defaults-from /Common/http
    interval 5
    ip-dscp 0
    recv none
    recv-disable none
    send "GET /\r\n"
    time-until-up 0
    timeout 16
}
```
converted as3 json

```
{
    "class": "ADC",
    "schemaVersion": "3.31.0",
    "id": "urn:uuid:54070ac9-1c55-429e-b5d5-c63c8060e9e5",
    "label": "Converted Declaration",
    "remark": "Generated by Automation Config Converter",
    "github75": {
        "class": "Tenant",
        "Shared": {
            "class": "Application",
            "template": "shared",
            "acc_test_http_monitor": {
                "adaptive": false,
                "interval": 5,
                "dscp": 0,
                "send": "GET /\\r\\n",
                "timeUntilUp": 0,
                "timeout": 16,
                "class": "Monitor",
                "monitorType": "http"
            }
        }
    }
}
```
Since TMSH is inputting none, ACC doesn't make any modifications

Here is the default

```
"acc_test_http_monitor": {
                "adaptive": false,
                "interval": 5,
                "dscp": 0,
                "send": "GET /\\r\\n",
                "timeUntilUp": 0,
                "timeout": 16,
                "class": "Monitor",
                "monitorType": "http",
                "targetAddress": "",
                "upInterval": 0,
                "targetPort": 0,
                "receive": "HTTP/1.",  ----------------- default
                "receiveDown": "",
                "reverse": false,
                "transparent": false,
                "adaptiveDivergenceType": "relative",
                "adaptiveDivergenceMilliseconds": 500,
                "adaptiveDivergencePercentage": 100,
                "adaptiveLimitMilliseconds": 1000,
                "adaptiveWindow": 180
            },
```

**Comment by mdditt2000** [comment]
![image](https://user-images.githubusercontent.com/12733938/163597558-36cb227e-396a-4541-9e4f-36ad60ecd713.png)

https://github.com/mdditt2000/f5-appsvcs-acc/blob/master/Github/75/as3-default.json
 
default "receive": "HTTP/1.",
 
Changing “recv none” to “receive”: “” is a schema change.

**Comment by mdditt2000** [comment]
re-opening

**Comment by p-semenov-f5** [comment]
Fixed in 1.21
